### PR TITLE
Forbid Client Account Creation

### PR DESCRIPTION
**Pull request by reustle**
It looks like it was already partially implemented, but I finished it up and added a config param / readme note.

Would like to hear your thoughts on the new readme text, I think it can be improved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,14 +118,32 @@ accountsUIBootstrap3.setCustomSignupOptions = function() {
...
+
+If you would like to prevent users from creating new accounts, you can choose to not show the account creation buttons as shown below.
+
+```javascript
+Accounts.ui.config({
+    forbidClientAccountCreation : true
+});
+```
+
+While this flag prevents the user from seeing the "Create Account" link, the server will still accept new accounts. You can prevent new account creation altogether by always returning false in the [account validation stage](http://docs.meteor.com/#/full/accounts_validatenewuser) on the server. Here is an example (place in server-side code):
```
Comment (ianmartorell) [inline README.md]: Actually, this is not true. If you run the `Accounts.ui.config` method both on the client and the server, it won't accept new accounts either, even if you try to do it through the browser console.

Comment (reustle) [inline README.md]: Oh great, then I'll scrap this bit.

**Comment by ianmartorell** [comment]
Alright, my comment was not completely correct. The thing is this feature already exists and it's in `Accounts.config`, not `Accounts.ui.config` (hence my confusion). See the docs [here](http://docs.meteor.com/#/full/accounts_config). So basically there is no need to implement it in this package, because setting the `forbidAccountCreation` flag like so already blocks new user creation _and_ hides the related dialogs and buttons:

```javascript
Meteor.startup(function() {
	Accounts.config({
		forbidClientAccountCreation : true
	})
})
```

**Comment by reustle** [comment]
Oh, that definitely clears things up. Thanks!

**Pull request closed without merging**